### ISSUE TITLE
[1.1 beta] /charban Doesn't Work

### ISSUE THREAD

**Issue by Pilot1000** [rebel1324/NutScript#266]
The command /charban doesn't delete a character. It doesn't return any lua error's either.

**Comment by brianhang** [comment]
It isn't suppose to delete a character. It sets the character as banned so the player cannot use the character.

**Comment by Pilot1000** [comment]
Right, I didn't write that correctly. But it doesn't do any of that. Doesn't give a notification, kick the player off of their character, or stop them from using it after going back to the character menu.

**Comment by brianhang** [comment]
Looks like there was a Lua error when sending the notification from the server.

**Comment by brianhang** [comment]
This should be fixed by a2d7fea61ae157ffaa55fa5d8b060659593d2088.
